docs/archive/api/sql/GeoSparkSQL-Predicate.md

```diff
@@ -100,4 +100,19 @@ Spark SQL example:
 SELECT *
 FROM geom
 WHERE ST_Overlaps(geom.geom_a, geom.geom_b)
+```
+
+## ST_Disjoint
```

Comment:
You should NOT make change to this doc. This is an archived GeoSparkSQL doc. Please make it in the current SQL doc.